domain/schema/controller.go

```diff
@@ -482,3 +485,65 @@ BEGIN
     VALUES (4, 5, OLD.key, DATETIME('now'));
 END;`)
 }
+
+func upgradeInfoSchema() database.Delta {
+	return database.MakeDelta(`
+CREATE TABLE upgrade_info (
+    uuid             TEXT PRIMARY KEY,
+    previous_version TEXT NOT NULL,
+    target_version   TEXT NOT NULL,
+    created_at       TIMESTAMP NOT NULL,
+    started_at       TIMESTAMP,
+    completed_at     TIMESTAMP
+);
+
+CREATE TABLE upgrade_node_status (
+    id     INT PRIMARY KEY,
+    status TEXT NOT NULL
+);
+
+INSERT INTO upgrade_node_status VALUES
```

Comment:
why are we creating a collection to hold constants?

Reply:
See below